certified-connectors/AIForged/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4610 @@
+        },
+        "userName": {
+          "type": "string"
+        },
+        "serviceId": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "serviceDocId": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "provider": {
+          "type": "string"
+        },
+        "settingId": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "workItem": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "transactionId": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "charge": {
+          "type": "number",
+          "format": "double"
+        },
+        "paramDefId": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "paramDefName": {
+          "type": "string"
+        },
+        "projectId": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "projectName": {
+          "type": "string"
+        },
+        "docId": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "docFileName": {
+          "type": "string"
+        },
+        "docContentType": {
+          "type": "string"
+        },
+        "docUsage": {
+          "$ref": "#/definitions/AIForged.DAL.UsageType"
+        },
+        "docStatus": {
+          "$ref": "#/definitions/AIForged.DAL.DocumentStatus"
+        },
+        "classId": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "className": {
+          "type": "string"
+        },
+        "year": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "month": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "day": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "dayOfWeek": {
+          "$ref": "#/definitions/System.DayOfWeek"
+        },
+        "count": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "average": {
+          "type": "number",
+          "format": "float"
+        },
+        "min": {
+          "type": "number",
+          "format": "float"
+        },
+        "max": {
+          "type": "number",
+          "format": "float"
+        }
+      }
+    },
+    "System.DayOfWeek": {
+      "type": "integer",
+      "description": "",
+      "x-enumNames": [
+        "Sunday",
+        "Monday",
+        "Tuesday",
+        "Wednesday",
+        "Thursday",
+        "Friday",
+        "Saturday"
+      ],
+      "enum": [
+        0,
+        1,
+        2,
+        3,
+        4,
+        5,
+        6
+      ]
+    }
+  },
+  "securityDefinitions": {
+    "oauth2": {
+      "type": "oauth2",
+      "flow": "accessCode",
+      "authorizationUrl": "https://portal.aiforged.com/connect/authorize",
+      "tokenUrl": "https://portal.aiforged.com/connect/token",
+      "scopes": {
+        "openid": "",
```

Comment:
@aiforged Empty property values can be removed until unless you have a specific reason to keep them.

Reply:
added property value in latest commit